lib/temporal.rb

```diff
@@ -36,6 +36,18 @@ def start_workflow(workflow, *input, **args)
       response.run_id
     end
 
+    def terminate_workflow(workflow, workflow_id:, **options)
```

Comment:
Can you please make it look like https://github.com/coinbase/cadence-ruby/pull/19?

Reply:
Do you mean the method signature? I think it should line up with start_workflow/schedule_workflow, and they use kwargs. `terminate_workflow_execution` takes namespace as an argument so I think ExecutionOptions should be used so that the Workflow classes namespace configuration is taken into account (as for start/schedule).

Reply:
Updated the method signature, although left `namespace` as nillable because it has a default value we can use.